packages/svelte/src/internal/client/reactivity/deriveds.js

```diff
@@ -104,6 +108,21 @@ function destroy_derived_children(derived) {
  */
 let stack = [];
 
+/**
+ * @param {Derived} derived
+ * @returns {Effect | null}
+ */
+export function get_derived_parent_effect(derived) {
+	var parent = derived.parent;
+	if (parent !== null) {
```

Comment:
Doesn't this need to be a loop in case it's nested several levels deep?
```suggestion
	while (parent !== null) {
```